tests/tests/classes/Serverless.js

```diff
@@ -8,108 +8,14 @@ const expect = require('chai').expect;
 const Serverless = require('../../../lib/Serverless');
 
 describe('Serverless', () => {
-  let serverless;
 
-  beforeEach(() => {
-    serverless = new Serverless({
-      interactive: false,
-    });
-  });
-
-  describe('#addPlugin()', () => {
-    it('should add plugins', () => {
-      class PluginMock {
-        constructor() {
-          this.commands = {};
-        }
-      }
-
-      serverless.addPlugin(PluginMock);
+  describe('#getVersion()', () => {
 
-      expect(serverless._plugins.length).to.be.equal(1);
-      expect(typeof serverless._plugins[0]).to.be.equal('object');
+    it('should get the correct serverless version', () => {
+      const serverless = new Serverless();
+      expect(serverless.getVersion()).to.be.equal('0.5.5');
```

Comment:
This test will break whenever we update the version. Maybe we simply test that it follows a specific regex, e.g.

```
expect(serverless.getVersion()).to.match(/\d+\.\d+\.\d+/);
```